src/stylesheets/components/_card.scss

```diff
@@ -4,6 +4,14 @@
   border-radius: 4px;
   background-color: white;
   box-shadow: 0 10px 25px 5px rgba(0, 0, 0, 0.15);
+
+  h2 {
+    margin: 10px 0;
+  }
+
+  a:visited {
+    color: #3f57a6;
```

Comment:
Prefer to use the token value and mixin/utils to grab color, background color, margin, padding, etc.
(see https://designsystem.digital.gov/utilities/ for all available)